package.json

```diff
@@ -21,7 +21,6 @@
     "lint": "eslint --fix --ext .ts ./src",
     "docs": "typedoc --options typedoc.js src",
     "prepack": "npm run build",
-    "prepublishOnly": "npm test && npm run lint",
```

Comment:
tests are being run during the `build` job. But the lint wasn't. Added that command to the job and removed this hook, so they are not re-run in the new job.